app/common/model/src/main/java/io/syndesis/common/model/Kind.java

```diff
@@ -56,6 +56,7 @@
     Permission(io.syndesis.common.model.user.Permission.class),
     Role(io.syndesis.common.model.user.Role.class),
     User(io.syndesis.common.model.user.User.class),
+    Quota(io.syndesis.common.model.user.Quota.class),
```

Comment:
do we need this, I kinda thought it was only needed for DB access

Reply:
It's needed to generate the Immutable.